src/cascade/input_data/db/asdr.py

```diff
@@ -97,6 +87,13 @@ def _upload_asdr_data_to_tier_3(cursor, model_version_id, asdr_data):
         )
     """
 
+    age_group_data = db_queries.get_age_metadata(
+        age_group_set_id=AGE_GROUP_SET_ID, gbd_round_id=execution_context.parameters.gbd_round_id
+    )[["age_group_id", "age_group_years_start", "age_group_years_end"]]
+
+    age_group_data.columns = ["age_group_id", "age_lower", "age_upper"]
```

Comment:
If the order of the columns doesn't match this, does it work OK? I'd feel more comfortable using `rename({}, axis=1)`

Reply:
It is guaranteed to match because of the line above but I agree, rename is better.